x/authority/types/authorizations.go

```diff
@@ -0,0 +1,129 @@
+		"/zetachain.zetacore.observer.MsgEnableCCTX",
+		"/zetachain.zetacore.observer.MsgUpdateGasPriceIncreaseFlags",
+		"/zetachain.zetacore.lightclient.MsgEnableHeaderVerification",
+	}
+	AdminPolicyMessages = []string{
+		"/zetachain.zetacore.crosschain.MsgMigrateTssFunds",
+		"/zetachain.zetacore.crosschain.MsgUpdateTssAddress",
+		"/zetachain.zetacore.fungible.MsgUpdateContractBytecode",
+		"/zetachain.zetacore.fungible.MsgUpdateSystemContract",
+		"/zetachain.zetacore.observer.MsgUpdateObserver",
+	}
+	EmergencyPolicyMessages = []string{
+		"/zetachain.zetacore.crosschain.MsgAddInboundTracker",
+		"/zetachain.zetacore.crosschain.MsgAddOutboundTracker",
+		"/zetachain.zetacore.crosschain.MsgRemoveOutboundTracker",
+		"/zetachain.zetacore.fungible.MsgPauseZRC20",
+		"/zetachain.zetacore.observer.MsgUpdateKeygen",
+		"/zetachain.zetacore.observer.MsgDisableCCTX",
+		"/zetachain.zetacore.lightclient.MsgDisableHeaderVerification",
+	}
+)
+
+// DefaultAuthorizationsList list is the list of authorizations that presently exist in the system.
+// This is the minimum set of authorizations that are required to be set when the authorization table is deployed
+func DefaultAuthorizationsList() AuthorizationList {
+	authorizations := make(
+		[]Authorization,
+		len(OperationPolicyMessages)+len(AdminPolicyMessages)+len(EmergencyPolicyMessages),
+	)
+	index := 0
+	for _, msgURL := range OperationPolicyMessages {
+		authorizations[index] = Authorization{
+			MsgUrl:           msgURL,
+			AuthorizedPolicy: PolicyType_groupOperational,
+		}
+		index++
+	}
+	for _, msgURL := range AdminPolicyMessages {
+		authorizations[index] = Authorization{
+			MsgUrl:           msgURL,
+			AuthorizedPolicy: PolicyType_groupAdmin,
+		}
+		index++
+	}
+	for _, msgURL := range EmergencyPolicyMessages {
+		authorizations[index] = Authorization{
+			MsgUrl:           msgURL,
+			AuthorizedPolicy: PolicyType_groupEmergency,
+		}
+		index++
+	}
+
+	return AuthorizationList{
+		Authorizations: authorizations,
+	}
+}
+
+// SetAuthorizations adds the authorization to the list. If the authorization already exists, it updates the policy.
+func (a *AuthorizationList) SetAuthorizations(authorization Authorization) {
+	for i, auth := range a.Authorizations {
+		if auth.MsgUrl == authorization.MsgUrl {
+			a.Authorizations[i].AuthorizedPolicy = authorization.AuthorizedPolicy
+			return
+		}
+	}
+	a.Authorizations = append(a.Authorizations, authorization)
+}
+
+// RemoveAuthorizations removes the authorization from the list. It does not check if the authorization exists or not.
+func (a *AuthorizationList) RemoveAuthorizations(authorization Authorization) {
+	for i, auth := range a.Authorizations {
+		if auth.MsgUrl == authorization.MsgUrl {
+			a.Authorizations = append(a.Authorizations[:i], a.Authorizations[i+1:]...)
+		}
+	}
+}
+
+// GetAuthorizedPolicy returns the policy for the given message url.If the message url is not found,
+// it returns an error and the first value of the enum.
```

Comment:
```
and the first value of the enum.
```
This comment is not necessary. When a function returns an error, all the other values should not be considered by the calling function, so its value doesn't matter